Source/WebKit/Platform/IPC/cocoa/DaemonConnectionCocoa.mm

```diff
@@ -39,6 +39,8 @@
 void Connection::send(xpc_object_t message) const
 {
     ASSERT(RunLoop::isMain());
+    initializeConnectionIfNeeded();
```

Comment:
Given that there is `ASSERT(m_connection.get());`, it seems this change is expected